ecs-cli/Gopkg.toml

```diff
@@ -65,7 +65,7 @@
 
 [[constraint]]
   name = "github.com/stretchr/testify"
-  version = "1.2.1"
+  version = "=1.2.1"
```

Comment:
I pinned it down to a specific version number that we know that works, instead of the default that will use a higher patch version if it exists (see behavior in https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#version).
We previously ran into an issue before where running `dep ensure -update` on the aws-sdk-go resulted in a broken build 😝. By pinning it we know what we get.

Upgraded to "=1.3.0"

Reply:
Ah ha! Didn't realize that's what broke us on the aws-sdk-go build before.